File: php-breeds/app/models/SpecieFish.php
<?php

Class SpecieFish {
	private $id;
	private $name;
	private $name_es;
	private $description;
	private $url_wiki;
	private $url_image;
	private $morphology;
	private $habitat;
	private $feeding;
	private $phylum;
	private $class;
	private $orden;
	private $family;
	private $genus;
	private $path_image;
	private $date;
	private $creator_id;
	
	/**
	 * CONSTRUCTOR
	 */
	function __construct($id){
		$this->id=$id;
	}
	

	/**
	 * SETTERS
	 */
	function set_name($name){
		$this->name=$name;
	}
    function set_name_es($name_es){
		$this->name_es=$name_es;
	}
    function set_description($description){
		$this->description=$description;
	}
	function set_url_wiki($url_wiki){
		$this->url_wiki=$url_wiki;
	}
	function set_url_image($url_image){
		$this->url_image=$url_image;
	}
	function set_morphology($morphology){
		$this->morphology=$morphology;
	}
	function set_habitat($habitat){
		$this->habitat=$habitat;
	}
	function set_feeding($feeding){
		$this->feeding=$feeding;
	}
	function set_phylum($phylum){
		$this->phylum=$phylum;
	}
	function set_class($class){
		$this->class=$class;
	}
	function set_orden($orden){
		$this->orden=$orden;
	}
	function set_family($family){
		$this->family=$family;
	}
	function set_genus($genus){
		$this->genus=$genus;
	}
    function set_path_image($path_image){
		$this->path_image=$path_image;
	}
    function set_date($date){
		$this->date=$date;
	}
    function set_creator_id($creator_id){
		$this->creator_id=$creator_id;
	}
    /**
	 * GETTERS
	 */
	function get_id(){
		return $this->id;
	}
	function get_name(){
		return $this->name;
	}
    function get_name_es(){
		return $this->name_es;
	}
    function get_description(){
		return $this->description;
	}
	function get_url_wiki(){
		return $this->url_wiki;
	}
	function get_url_image(){
		return $this->url_image;
	}
	function get_morphology(){
		return $this->morphology;
	}
	function get_habitat(){
		return $this->habitat;
	}
	function get_feeding(){
		return $this->feeding;
	}
	function get_phylum(){
		return $this->phylum;
	}
	function get_class(){
		return $this->class;
	}
	function get_orden(){
		return $this->orden;
	}
	function get_family(){
		return $this->family;
	}
	function get_genus(){
		return $this->genus;
	}
    function get_path_image(){
		return $this->path_image;
	}
    function get_date(){
		return $this->date;
	}
    function get_creator_id(){
		return $this->creator_id;
	}

    	
	/**
	 * TODTRING
	 */
	function to_String(){
		return"<br><img src=".PATHSERVER.$this->path_image." width=40px>Cat id: ".$this->id.", name: ".$this->name.", path: ".$this->path_image.", creator: ".$this->creator_id;	
	}
}

?>

File: java-ConsumoAPI/src/main/java/com/mycompany/consumoapi/models/SpecieFish.java
package com.mycompany.consumoapi.models;

/**
 *
 * @author kikem
 */
public class SpecieFish {
    private int id;
    private String name;
    private String name_es;
    private String description;
    private String urlWiki;
    private String url_image;
    private String path_image;
    private String date;
    private int creator_id;

    public SpecieFish() {
    }

    public int getId() {
        return id;
    }

    public SpecieFish(int id, String name, String name_es, String description, String urlWiki, String url_image,String path_image, String date, int creator_id) {
        this.id = id;
        this.name = name;
        this.name_es = name_es;
        this.description = description;
        this.urlWiki = urlWiki;
        this.url_image = url_image;
        this.path_image = path_image;
        this.date = date;
        this.creator_id = creator_id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName_es() {
        return name_es;
    }

    public void setName_es(String name_es) {
        this.name_es = name_es;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getUrlWiki() {
        return urlWiki;
    }

    public void setUrlWiki(String urlWiki) {
        this.urlWiki = urlWiki;
    }
    
    
    
    public String getURL_image() {
        return url_image;
    }

    public void setURL_image(String url_image) {
        this.url_image = url_image;
    }
    
    
    public String getPath_image() {
        return path_image;
    }
    public void setPath_image(String path_image) {
        this.path_image = path_image;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public int getCreator_id() {
        return creator_id;
    }

    public void setCreator_id(int creator_id) {
        this.creator_id = creator_id;
    }

    @Override
    public String toString() {
        return "SpecieFish{" + "id=" + id + ", name=" + name + ", name_es=" + name_es + ", description=" + description + ", urlWiki=" + urlWiki + ", path_image=" + path_image + ", date=" + date + ", creator_id=" + creator_id + '}';
    }

  
    
    
}
